### PR TITLE
Clarify behaviors and timelines in WEBGL_lose_context.

### PR DESCRIPTION
**Pull request by jdashg**
This resolves ambiguities discovered in
https://bugzilla.mozilla.org/show_bug.cgi?id=1144115

### CODE REVIEW

File: extensions/WEBGL_lose_context/extension.xml [hunk 1]
```diff
@@ -52,14 +52,30 @@ interface WEBGL_lose_context {
...
       the conditions in the WebGL specification for restoring the
       context are not satisfied, or if the context was not lost via <code>loseContext()</code>,
       generate an <code>INVALID_OPERATION</code> error.
+      </p>
+
+      <p>
+      For example, if <code>e.preventDefault()</code> is not called on the 'webglcontextlost'
+      event, the context will not be marked restorable, and <code>restoreContext()</code> will fail.
+      Additionally, the context is not restorable before the 'webglcontextlost' event is complete:
+      Calling <code>restoreContext()</code> before 'webglcontextlost' has completed will fail.
```
Comment (kenrussell) [inline extensions/WEBGL_lose_context/extension.xml]: Calling -> calling

File: extensions/WEBGL_lose_context/extension.xml [hunk 2]
```diff
@@ -52,14 +52,30 @@ interface WEBGL_lose_context {
...
+      </p>
     </function>
   </newfun>
   <history>
```
Comment (kenrussell) [inline extensions/WEBGL_lose_context/extension.xml]: Would you please add a revision entry below?

File: extensions/WEBGL_lose_context/extension.xml (hunk 1 above)
Comment (jdashg) [reply extensions/WEBGL_lose_context/extension.xml]: Wow, but I think you're right! :)

**Comment by jdashg** [comment]
I don't know if we test all these things directly.

**Pull request merged**